src/Mod/Part/App/TopoShape.cpp

```diff
@@ -2503,6 +2503,62 @@ TopoDS_Shape TopoShape::makeLongHelix(Standard_Real pitch, Standard_Real height,
     return TopoDS_Shape(std::move(wire));
 }
 
+TopoDS_Shape TopoShape::makeSpiralHelix(Standard_Real radiusbottom, Standard_Real radiustop,

```

Comment:
makeLongHelix, returns a TopoShape. Which could be expected from a method in the TopoShape class. Personally I almost always want the TopoDS_Shape, so I don't really mind.

Reply:
No, makeHelix() and makeLongHelix() return TopoDS_Shape already.
I copy/pasted one of these two functions as a skeleton to make the new one.

Reply:
Ok, I checked again and you are right. Don't know what I was talking about.